COLLABORATOR_GUIDE.md

```diff
@@ -263,6 +263,20 @@ multiple commits. Commit metadata and the reason for the revert should be
 appended. Commit message rules about line length and subsystem can be ignored.
 A Pull Request should be raised and approved like any other change.
 
+### Introducing New Modules
+
+Semver-minor commits that introduce new core modules should be treated with
+extra care.
+
+* First, the name of the new core module should not conflict with any existing
```

Comment:
I assume there should be an email, GitHub comment, or some other paper trail that shows the agreement with the existing module owner?

Reply:
something along those lines yes.. I'm not sure how prescriptive we need to be with it.